src/main/scala/temple/generate/utils/CodeTerm.scala

```diff
@@ -20,12 +20,16 @@ trait CodeTerm {
 
 object CodeTerm {
 
-  /** Turns a [[String]] into a [[CodeTerm]], with a trivial iterator of just the single string
-    */
+  /** Turns a [[String]] into a [[CodeTerm]], with a trivial iterator of just the single string */
   implicit class CodeTermString(string: String) extends CodeTerm {
     override def flatIterator: Iterator[String] = Iterable.single(string).iterator
   }
 
+  /** Turns a [[None]] into a [[CodeTerm]], with a trivial iterator of nothing */
+  implicit class CodeTermNone(none: None.type) extends CodeTerm {
```

Comment:
Can we not use `Optional` here, flatten out the `None`? Feels like it could be useful - similar to the JSON stuff